services/QuillLMS/app/models/classroom_unit.rb

```diff
@@ -38,6 +38,7 @@ class ClassroomUnit < ApplicationRecord
   validates :unit, uniqueness: { scope: :classroom }
   before_save :check_for_assign_on_join_and_update_students_array_if_true
   after_save  :hide_appropriate_activity_sessions
+  after_touch :touch_classroom_without_callbacks
```

Comment:
If we're doing `after_save` and `after_touch`, I think we could just do one `after_commit` instead (might be worth confirming that).

Reply:
That seems reasonable.  Let me do some testing on that front to confirm.